extension/src/test/fixtures/plotsDiff/index.ts

```diff
@@ -936,10 +1004,60 @@ const getIndFromComparisonMultiImgPath = (path: string) => {
   return Number((pathIndMatches as string[])[1])
 }
 
+const boundingBoxColors = ['#ff3838', '#ff9d97', '#ff701f']
+
+export const collectPlotClasses = ({
+  plotClasses,
+  classLabels,
+  boundingBoxes,
+  id,
+  path
+}: {
+  plotClasses: ComparisonPlotClasses
+  classLabels: Set<string>
+  boundingBoxes: { label: string; box: BoundingBox }[]
+  id: string
+  path: string
+}) => {
+  const classAcc: {
+    [label: string]: {
+      label: string
+      boxes: {
+        h: number
+        w: number
+        x: number
+        y: number
+      }[]
+    }
+  } = {}
+
+  for (const { label, box } of boundingBoxes) {
+    classLabels.add(label)
+
+    if (!classAcc[label]) {
+      classAcc[label] = { label, boxes: [] }
+    }
+
+    classAcc[label].boxes.push({
+      x: box.left,
+      y: box.top,
+      w: box.right - box.left,
+      h: box.bottom - box.top
```

Comment:
Isn't this doing the same as [this](https://github.com/iterative/vscode-dvc/pull/5250/files#diff-b8591ca0f7ff9633208614960ef48ca0fd0bcc1fb055531bbf943a8250a14d86R503)

Reply:
Yes, it is doing the same thing. Is there another way to about this? I think we want to keep the webview message creation logic separate from our extension's collect logic?

Reply:
Done!

Reply:
Can we leave left, right, top and bottom as variable names for as long as possible in the system? Also, how is x box.left?

Reply:
> Can we leave left, right, top and bottom as variable names for as long as possible in the system?

I believe we're [already doing that ](https://github.com/iterative/vscode-dvc/pull/5250/files#diff-9602466799fe1c9d6f9748ba405f6e319440fcaccc9473a1fcf274c133868adbL63). We currently use left, right, top and bottom terms throughout the frontend and the backend, getting x, y, width, height when we are creating the SVGs. Unless I'm misunderstanding what you mean?